lib/modules/manager/github-actions/readme.md

```diff
@@ -40,3 +40,6 @@ jobs:
   build:
     runs-on: ${{ env.RUNNER }}
 ```
+
+The `github-action` manager also supports `ratchet` comments, like `# ratchet:actions/checkout@v2.1.0`.
```

Comment:
```suggestion
The `github-action` manager understands `ratchet` comments, like `# ratchet:actions/checkout@v2.1.0`.
This means that Renovate will:

- update the version of a _pinned_ Ratchet version if needed
- not delete/modify Ratchet comments after parsing them
- keep `# ratchet:exclude` comments

```

> I mean the ratchet comments with pinned versions would be taken into consideration as well as `# pin @2.1.0` comments. This also means that ratchet comments don't get deleted/modified after being parsed. It is also the case, that `# ratchet:exclude` comments would not be deleted.

I think a short list of the supported usecases/behavior would help understand what's going to happen.

Please review my suggestion for accuracy before applying it. :wink: